frontend/www/src/js/modules/dashboard/tradition/propertyTableView.js

```diff
@@ -160,7 +217,9 @@ class PropertyTableView extends HTMLElement {
           ${metaItems.map(this.renderMetaItem).join('\n')}
           </tbody>
         </table>
+        <another-button/>
```

Comment:
Is this just a placeholder for now?

Reply:
:-) I have no idea what this does here, probably a remnant of met toying around. I'll remove it with the next PR.